### DISCUSSION TITLE
VIP only methods

### DISCUSSION THREAD

**Discussion by rectifyer** [trakt/trakt-api#303] [Announcements]
We’re introducing some VIP only methods. This allows your apps to offer some of the VIP only website functionality. https://trakt.docs.apiary.io/#introduction/vip-only-methods

**First up:** resetting show watched progress (and undoing it). https://trakt.docs.apiary.io/#reference/shows/reset-watched-progress/reset-show-progress

**Comment by eliehaykal** [comment]
That is quite awesome.. looking forward to more methods :)

**Comment by juanma-cvega** [comment]
Hi, are there any other methods being added to the list? I haven't seen any news on this. I'm particularly interested in the in progress TV shows per user list one. I actually signed up for the VIP membership for this feature.

**Reply by rectifyer** [reply]
> progress TV shows per user list one

What do you mean by this? Do you have an example on the Trakt website?

**Reply by juanma-cvega** [reply]
This is an example from my account.
https://trakt.tv/users/sepharat/progress?list=3281951
So basically I would like to know if there is any way to achieve this via the API.
